internal/operator/builder.go

```diff
@@ -219,6 +220,18 @@ func (b *Builder) Build(ctx context.Context) (cluster.Cluster, error) {
 		return nil, fmt.Errorf("unable to create indexers: %w", err)
 	}
 
+	npeReconciler := atlasnetworkpeering.NewAtlasNetworkPeeringsReconciler(
```

Comment:
blocking: this needs to be in the controller registry https://github.com/mongodb/mongodb-atlas-kubernetes/blob/main/internal/controller/registry.go

Reply:
Right! yep I also need to fix this for comply with the new **Cluster** support